presto-hive/src/main/java/com/facebook/presto/hive/metastore/ThriftHiveMetastore.java

```diff
@@ -769,6 +737,38 @@ public void revokeTablePrivileges(String databaseName, String tableName, String
         }
     }
 
+    private PrivilegeBag buildPrivilegeBag(HiveMetastoreClient metastoreClient,
+                                           String databaseName,
+                                           String tableName,
+                                           String grantee,
+                                           Set<PrivilegeGrantInfo> privilegeGrantInfos)
```

Comment:
formatting is still off... the arguments should be indented twice, and the first argument should be on a new line